models/event/event.go

```diff
@@ -69,6 +69,7 @@ func ToMemoryEvent(event cdc.Event, pkMap map[string]any, tc *kafkalib.TopicConf
 
 		// We don't need this either.
 		delete(evtData, constants.DeleteColumnMarker)
+		delete(evtData, constants.OnlySetDeleteColumnMarker)
```

Comment:
This removes the marker if we're in history mode, so that each dialect's `Append()` method doesn't have to filter it out like the `Merge()` methods do.

Reply:
Can we add a test to this to ensure the column is being deleted? Can be a FF